shell/common/switches.h

```diff
@@ -104,7 +104,6 @@ DEF_SWITCH(DartNonCheckedMode,
            "precompiled and checked mode is unsupported. However, this flag "
            "may be specified if the user wishes to run in the debug product "
            "mode (i.e. with JIT or DBC) with checked mode off.")
-DEF_SWITCH(DartStrongMode, "strong", "Enable Dart 2.0 strong mode.")
```

Comment:
Do we need to fix all the places where --strong is passed(like https://github.com/flutter/engine/blob/master/shell/platform/android/io/flutter/app/FlutterActivityDelegate.java#L305 for example)

Reply:
I removed it from FlutterActivityDelegate.java see file below.